src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -14,6 +14,12 @@ define(function (require) {
     var defaultMapCenter = [15, 5];
     var defaultMapZoom = 2;
 
+    // Convenience function to turn around the LngLat recieved from ES
```

Comment:
This is duplicated here: https://github.com/panda01/kibana/blob/fix/3810/fitMapDataBounds/src/kibana/components/vislib/visualizations/tile_map.js#L217

Can you remove that instance of it?